site/docs/try/configuration.md

```diff
@@ -55,19 +55,26 @@ Starting with Nessie 0.66.0, tracing and metrics are always used, if OpenTelemet
+| `nessie.version.store.persist.bigtable.instance-id`           | `nessie`       | `String`              | Sets the instance-id to be used with Google BigTable.                                                                    |
+| `nessie.version.store.persist.bigtable.quota-project-id`      | n/a            | `String`              | Google BigTable quote project ID (optional).                                                                             |
+| `nessie.version.store.persist.bigtable.emulator-host`         | n/a            | `String`              | When using the BigTable emulator, used to configure the host.                                                            |
+| `nessie.version.store.persist.bigtable.emulator-port`         | `8086`         | `int`                 | When using the BigTable emulator, used to configure the port.                                                            |
+| `nessie.version.store.persist.bigtable.endpoint`              | n/a            | `String`              | Google BigTable endpoint (if not default).                                                                               |
+| `nessie.version.store.persist.bigtable.mtls-endpoint`         | n/a            | `String`              | Google BigTable MTLS endpoint (if not default).                                                                          |
+| `nessie.version.store.persist.bigtable.jwt-audience-mapping`  | n/a            | `Map<String, String>` | Google BigTable JWT audience mappings (if necessary).                                                                    |
+| `nessie.version.store.persist.bigtable.table-prefix`          | n/a            | `String`              | Prefix for tables, default is no prefix.                                                                                 |
+| `nessie.version.store.persist.bigtable.min-channel-count`     | `1`            | `int`                 | Minimum number of gRPC channels. Refer to Google docs for details.                                                       |
+| `nessie.version.store.persist.bigtable.max-channel-count`     | `1`            | `int`                 | Maximum number of gRPC channels. Refer to Google docs for details.                                                       |
+| `nessie.version.store.persist.bigtable.initial-channel-count` | `1`            | `int`                 | Initial number of gRPC channels. Refer to Google docs for details.                                                       |
+| `nessie.version.store.persist.bigtable.min-rpcs-per-channel`  | `1`            | `int`                 | Minimum number of RPCs per channel. Refer to Google docs for details.                                                    |
```

Comment:
Where does this default come from? `ChannelPoolSettings.builder().build()` appears to yield `0` for this property.

Reply:
Also `ChannelPoolSettings.builder().build().getMaxChannelCount()` is 200.